examples/mc.ipynb

```diff
@@ -124,32 +139,38 @@
     "\n",
     "    chi_max = 100.0\n",
     "\n",
-    "    # For every sample, we'll compute MT2 for this range of chi.\n",
+    "    # For every sample, we'll compute MT2 for this range of chi.\n",
     "    chis = numpy.arange(n_chi_bins + 1) * chi_max / n_chi_bins\n",
-    "    \n",
+    "\n",
     "    data_chi = []\n",
     "    data_mt2 = []\n",
     "    for _ in range(n_samples):\n",
-    "    \n",
-    "        parent1 = ran_four_mom(m_parent1 ** 2, numpy.random.uniform(10, 200))\n",
-    "        parent2 = ran_four_mom(m_parent2 ** 2, numpy.random.uniform(10, 200))\n",
+    "        parent1 = ran_four_mom(m_parent1**2, numpy.random.uniform(10, 200))\n",
+    "        parent2 = ran_four_mom(m_parent2**2, numpy.random.uniform(10, 200))\n",
     "\n",
     "        p1, i1 = decay(parent1, m_parent1, m_vis1, m_invis)\n",
     "        p2, i2 = decay(parent2, m_parent2, m_vis2, m_invis)\n",
     "\n",
     "        computed_mt2 = mt2(\n",
-    "            m_vis1, p1[0], p1[1],\n",
-    "            m_vis2, p2[0], p2[1],\n",
-    "            i1[0] + i2[0], i1[1] + i2[1],\n",
-    "            chis, chis)  # Symmetric configuration -- same mass for both invisible particles.\n",
+    "            m_vis1,\n",
+    "            p1[0],\n",
+    "            p1[1],\n",
+    "            m_vis2,\n",
+    "            p2[0],\n",
+    "            p2[1],\n",
+    "            i1[0] + i2[0],\n",
+    "            i1[1] + i2[1],\n",
+    "            chis,\n",
+    "            chis,\n",
+    "        )  # Symmetric configuration -- same mass for both invisible particles.\n",
```

Comment:
Move this comment to above chis

Reply:
https://github.com/tpgillam/mt2/pull/80/commits/a045c1d1b5fcad96898fcd5dd4d10a1a97d92361